include/triton/core/tritonserver.h

```diff
@@ -847,6 +857,34 @@ TRITONSERVER_DECLSPEC TRITONSERVER_Error* TRITONSERVER_InferenceTraceParentId(
 TRITONSERVER_DECLSPEC TRITONSERVER_Error* TRITONSERVER_InferenceTraceModelName(
     TRITONSERVER_InferenceTrace* trace, const char** model_name);
 
+/// Set the name associated with a trace.
+///
+/// \param trace The trace.
+/// \param name The name associated with the trace.
+/// \return a TRITONSERVER_Error indicating success or failure.
+TRITONSERVER_DECLSPEC TRITONSERVER_Error*
+TRITONSERVER_InferenceTraceSetModelName(
+    TRITONSERVER_InferenceTrace* trace, const char* name);
+
+/// Get the name associated with a trace. The caller does
+/// not own the returned string and must not modify or delete it. The
+/// lifetime of the returned string extends only as long as 'trace'.
+///
+/// \param trace The trace.
+/// \param name Returns the name associated with the trace.
+/// \return a TRITONSERVER_Error indicating success or failure.
+TRITONSERVER_DECLSPEC TRITONSERVER_Error* TRITONSERVER_InferenceTraceName(
+    TRITONSERVER_InferenceTrace* trace, const char** name);
```

Comment:
`trace_name` to match Setter API
```suggestion
/// \param trace_name Returns the name associated with the trace.
/// \return a TRITONSERVER_Error indicating success or failure.
TRITONSERVER_DECLSPEC TRITONSERVER_Error* TRITONSERVER_InferenceTraceName(
    TRITONSERVER_InferenceTrace* trace, const char** trace_name);
```